test/unit/modules.spec.js

```diff
@@ -925,4 +925,31 @@ describe('Modules', () => {
       /getters should be function but "getters\.test" in module "foo\.bar" is true/
     )
   })
+
+  it('module: computed getter should be reactive after module registration', () => {
```

Comment:
The test will fail when removing this line: https://github.com/vuejs/vuex/pull/2201/files#diff-d7ab881f3cc9bbff70c15e5574df58176996a0b834bc174d3f3cea7a64ad8a95R87 .